Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/git-issue-1958.dfy.expect

```diff
@@ -0,0 +1,19 @@
+git-issue-1958.dfy(13,16): Error: value does not satisfy the subset constraints of 'R'
+git-issue-1958.dfy(18,16): Error: value does not satisfy the subset constraints of 'R'
+git-issue-1958.dfy(32,4): Error: value does not satisfy the subset constraints of 'R'
+git-issue-1958.dfy(36,4): Error: value does not satisfy the subset constraints of 'R'
+git-issue-1958.dfy(56,16): Error: function precondition could not be proved
```

Comment:
That one, along with the one line 104, is not triggered on my machine and that's a soundness issue. So you fixed it !